draft-ietf-teep-protocol.md

```diff
@@ -1186,3 +1186,179 @@ ta-uuid = 16
 ta-manifest-sequence-number = 17
 have-binary = 18
 ~~~~
+
+# D. Examples of Diagnostic Notation and Binary Representation {#examples}
+{: numbered='no'}
+
+### Some assumptions in examples
+
+- TAM supports 2 versions: 0, 1
```

Comment:
Since version 1 is not defined, I don't think any example should use it

Reply:
Changed to use only version 0 at https://github.com/ietf-teep/teep-protocol/pull/65/commits/51e792b736ad20f114c17646802dee51c376e432